src/integrations/prefect-redis/prefect_redis/locking.py

```diff
@@ -34,15 +31,6 @@ def __init__(
         password: Optional[str] = None,
         ssl: bool = False,
     ) -> None:
-        """
-        Args:
-            host: The host of the Redis server
-            port: The port the Redis server is running on
-            db: The database to write to and read from
-            username: The username to use when connecting to the Redis server
-            password: The password to use when connecting to the Redis server
-            ssl: Whether to use SSL when connecting to the Redis server
-        """
```

Comment:
should we keep this?

Reply:
I removed it because it's a duplicate of the docstring on the class itself. Most classes don't have docstrings on `__init__` methods and instead defer to the class docstring.